content/en/docs/advanced/debug.md

```diff
@@ -3,9 +3,113 @@ draft: false
 linktitle: Enable Kubectl logs/exec to debug pods on the edge
 menu:
   docs:
-    parent: advanced configuration 
+    parent: advanced configuration
     weight: 30
 title: Enable Kubectl logs/exec to debug pods on the edge
 toc: true
 type: docs
 ---
+
+## Prepare certs
+
+1. Make sure you can find the kubernetes `ca.crt` and `ca.key` files. If you set up your kubernetes cluster by `kubeadm` , those files will be in `/etc/kubernetes/pki/` dir.
+
+    ``` shell
+    ls /etc/kubernetes/pki/
+    ```
+
+2. Set `CLOUDCOREIPS` env. The environment variable is set to specify the IP address of cloudcore, or a VIP if you have a highly available cluster.
+
+    ```bash
+    export CLOUDCOREIPS="192.168.0.139"
+    ```
+    (Warning: the same **terminal** is essential to continue the work, or it is necessary to type this command again.) Checking the environment variable with the following command:
+    ``` shell
+    echo $CLOUDCOREIPS
+    ```
+
+3. Generate the certificates for **CloudStream** on cloud node, however, the generation file is not in the `/etc/kubeedge/`, we need to copy it from the repository which was git cloned from GitHub.
+   Change user to root:
+    ```shell
+    sudo su
```

Comment:
What would happen if user don't use root?

Reply:
In this case, all cmds should start with `sudo` first.